exercise/0-IH-exercise/main.go

```diff
@@ -1,9 +1,54 @@
 package main
 
+import (
+	"fmt"
+	"runtime"
+	"time"
+)
+
+
 func main() {
-	// 3 task concurrent
-	// pertask beda2 sleepnya
-	// gunakan wait group dan channel
-	// waitnya tidak menunggu dari 5 detik
-	// cancelation bisa menggunakn time, context, channel
+	// use channel
+
+	runtime.GOMAXPROCS(2)
```

Comment:
Generally, we do not need to care about this. Goroutine abstracts num of core or CPU away. 
Unlike dealing with pure OS thread (such as forking in C maybe), co/goroutine make it possible for us to not care about how many possible threads available and focus on program logic. (do they have race conditions? deadlock? correct logic, etc.)